src/plugins/saved_objects/README.md

```diff
@@ -175,3 +175,656 @@ The migraton version will be saved as a `migrationVersion` attribute in the save
 ```
 
 For a more detailed explanation on the migration, refer to [`saved objects management`](src/core/server/saved_objects/migrations/README.md).
+
+## Server APIs
```

Comment:
Thank you for adding documentation for just not only for your changes but also all other APIs. Much appreciated!